src/java.base/share/classes/java/security/doc-files/debug-system-property.html

```diff
@@ -55,21 +55,6 @@ <h1><B>{@systemProperty java.security.debug}</B></h1>
     comma.
```

Comment:
I suggest we add a sentence to this paragraph noting that each trace message also includes the thread id, timestamp, and caller information.

Reply:
Thanks. Added 

"Each trace message includes the thread id, timestamp and caller information."  to this paragraph